test/app/smsController.js

```diff
@@ -0,0 +1,120 @@
+const assert = require('assert');
+const supertest = require('supertest');
+const should = require('should');
+
+const APP = require('../../app/app');
+
+const Emails = require('../../app/models/emails');
+const Capture = require('../../app/models/capture');
+const CommConns = require('../../app/models/commConns');
+const Communications = require('../../app/models/communications');
+const Conversations = require('../../app/models/conversations');
+const Messages = require('../../app/models/messages');
+
+const twilioAgent = supertest.agent(APP);
+const smsData = require('../testSmsData');
+
```

Comment:
Seems like good controller test coverage here. You have a bunch of well-defined model methods now that are used to build these controllers. Would ideally want to see model tests for those methods as well. These would arguably be easier to build/validate and could have been build in tandem with the refactor to test that methods were working without hitting the controller with a request. Tighter feedback loops, more granular test cases, etc....